### DISCUSSION TITLE
I want back to old style repo page

### DISCUSSION THREAD

**Discussion by turnon** [community/community#22120] [New to GitHub]
<p>seems new feature that the layout of repo page changes, showing two columns, README on the left and contributors, languege on the right.</p>
<p>I want back to old style repo page. How?</p>

**Comment by ghost** [comment]
<p>Hi <a class="mention" href="/turnon">@turnon</a>! Welcome to the GitHub Support Community!</p>
<p>While the new layout is in preview mode, you can toggle back to the original layout by going to Feature Preview in the profile dropdown menu:</p>
<p><img src="https://user-images.githubusercontent.com/10137/181068932-46229b82-8f6d-4fab-9d52-f40919cc358c.png" alt="preview-menu" data-base62-sha1="1Ro8aGXObPw6afe732nZcj782XP" width="214" height="493"></p>
<p>You can then disable and enable the Repository Refresh preview.</p>
<p>Of course if you have feedback about the new layout, we would love to hear it! You can let us know here:</p>
<ul>
<li><a href="https://support.github.com/contact/feedback">https://support.github.com/contact/feedback</a></li>
</ul>

**Comment by Landius** [comment]
<p>Can’t disable this feature now, new layout looks bad in wide screen.</p>

**Comment by Displee** [comment]
<p>It’s awe full. Please make a Chrome plugin to view Github in old style.</p>

**Comment by Real-Gecko** [comment]
<p>I tried and I did not like it, so I disabled it, but now I’m forced to use it. WTF? Gimme old layout back!</p>

**Comment by ghost** [comment, marked as answer]
<p><a class="mention" href="/real-gecko">@Real-Gecko</a> we have rolled the new layout out to all users, so it’s no longer possible to revert to the old version. If you do have specific feedback please let us know here:</p>
<p><a href="https://support.github.com/contact/feedback" class="onebox" target="_blank">https://support.github.com/contact/feedback</a></p>
